.github/workflows/ci.yml

```diff
@@ -11,17 +11,27 @@ jobs:
     strategy:
       matrix:
         java: [8, 11, 15]
+        kubernetes: ['v1.17.13','v1.18.12','v1.19.4']
     steps:
       - uses: actions/checkout@v2
       - uses: actions/cache@v2
         with:
           path: ~/.m2/repository
           key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
           restore-keys: |
-            ${{ runner.os }}-maven-  
+            ${{ runner.os }}-maven-
       - name: Set up JDK
         uses: actions/setup-java@v1
         with:
           java-version: ${{ matrix.java }}
-      - name: Build with Maven
-        run: mvn -B package -P no-integration-tests --file pom.xml
+      - name: Run unit tests
```

Comment:
Running unit tests before setting up the minikube help to get feedback from unit tests quicker and continue only if all unit tests passed